src/openrct2/ride/Vehicle.cpp

```diff
@@ -1539,7 +1539,7 @@ static void vehicle_update_measurements(rct_vehicle * vehicle)
     }
 
     uint8 stationId = ride->current_test_station;
-    if (ride->entrances[stationId].xy != RCT_XY8_UNDEFINED)
+    if (ride_get_entrance_location_of_station(vehicle->ride, stationId).x != LOCATION_NULL)
```

Comment:
More efficient here to pass "ride" as the first argument.

Reply:
You're right. There are reasons why I did it like this but those are no longer relevant.